frontend/packages/kubevirt-plugin/src/k8s/requests/v2v/start-v2vvmware-controller.ts

```diff
@@ -129,42 +136,57 @@ export const startV2VVMWareController = async (
   let kubevirtVmwareConfigMap = null;
 
   try {
-    kubevirtVmwareConfigMap = await getVmwareConfigMap({ k8sGet });
-
-    if (!kubevirtVmwareConfigMap) {
-      return; // pass through finally()
+    try {
+      kubevirtVmwareConfigMap = await getVmwareConfigMap();
+    } catch (error) {
+      if (error?.json.code === 403) {
+        throw new K8sDetailError({
```

Comment:
who catches the error ? isn't it the `catch` in line 169 ?

Reply:
no, that is just a catch to test the presence of the deployment. The error will get caught in vmware/ovirt provider actions and saved to redux.